### PR TITLE
macos-installer: bundle krunkit

### PR DESCRIPTION
**Pull request by slp**
Extend Makefile and package.sh to download, sign and bundle krunkit and its dependencies into the package.

#### Does this PR introduce a user-facing change?
```release-note
None
```

### CODE REVIEW

File: contrib/pkginstaller/Makefile [hunk 1]
```diff
@@ -8,8 +8,10 @@ else
 endif
 GVPROXY_VERSION ?= 0.7.3
 VFKIT_VERSION ?= 0.5.1
+KRUNKIT_VERSION ?= 0.1.0
 GVPROXY_RELEASE_URL ?= https://github.com/containers/gvisor-tap-vsock/releases/download/v$(GVPROXY_VERSION)/gvproxy-darwin
 VFKIT_RELEASE_URL ?= https://github.com/crc-org/vfkit/releases/download/v$(VFKIT_VERSION)/vfkit-unsigned
+KRUNKIT_RELEASE_URL ?= https://github.com/containers/krunkit/releases/download/v0.1.1/krunkit-podman-unsigned-0.1.1.tgz
```
Comment (rhatdan) [inline contrib/pkginstaller/Makefile:14]: Shouldn't this use KRUNKIT_VERSION rather then 0.1.1 And this version is 0.1.1 versus 0.1.0 above?

Comment (slp) [reply contrib/pkginstaller/Makefile]: Yes, I've put this draft in a haste and didn't pay much attention. Should be better now.

**Comment by rhatdan** [comment]
LGTM
@ashley-cui @lsm5 @baude @mheon PTAL

**Comment by lsm5** [comment]
CI is happy, and changes look fine so LGTM from me FWIW. But, I'll defer to @ashley-cui as I have no idea about macos installer.

**Review by ashley-cui — COMMENTED** [review]
Tested locally, everything seems to be in the right place! Thanks @slp 

/lgtm

**Pull request merged**